docs/index.md

```diff
@@ -1,24 +1,17 @@
-# Amplitude Analysis 101 / PWA101 (v2.0)
+# Symbolic Amplitude Models with $N^*$ Resonances at GlueX
 
-## CAS-assisted Amplitude Models building for photoproduction reaction
+We are doing amplitude analysis and Partial Wave Analysis (PWA) with [CAS-assisted model building](https://compwa.github.io/symbolics), which is a technique employed in the ComPWA project.
 
-This document is a follow-up of PWA101 (v1.0) (temporarily see [here](https://compwa--217.org.readthedocs.build/report/999.html)),
-as an introduction to Amplitude Analysis / Partial Wave Analysis (PWA) by demonstrating its application to a specific reaction channel and amplitude model.
+- One demonstration example is:<br>
+  $p \gamma \to$ Resonances (e.g. N\*) $\to \eta \pi^0 p$
+- The main focus of this research would be:<br>
+  $p \gamma \to$ Resonances (e.g. N\*, K\*) $\to K^+ \pi^0 \Lambda$
 
-Now the focus is the use of symbolic expressions (computations), with the use of [ComPWA packages](https://compwa.github.io/), or CAS(Computer Algebra System)-assisted model building in general (the python package `SymPy` in this case) to illustrate the general process in PWA.
-
-This tutorial aims to equip readers with deeper understanding of PWA methodologies and full workflow in the context of hadron physics through a practical, hands-on example with symbolic expressions via [ComPWA](https://compwa.github.io/).
-
-Furthermore,
-address to the issue ✅&nbsp;[ComPWA/gluex-nstar#1](https://github.com/ComPWA/gluex-nstar/issues/1), this document is PWA101(v2.0), which shows PWA methodologies and full workflow in the context of hadron physics with symbolic expressions via [ComPWA](https://compwa.github.io/).
-
-# Table of contents:
+This research project focuses on developing and applying amplitude models to analyze data from the GlueX experiment, specifically targeting $N^*$ and $K^*$ resonances.
 
 ```{toctree}
```

Comment:
The rendered table of content on the website landing page looks a bit cluttered. To avoid, set a `:maxdepth:`, with e.g. `2`.